create staging environment

ultrasaurus opened the pull request:
I wanted to drop and re-create staging database, and ran into a [Rails warning](https://blog.bigbinary.com/2016/06/07/rails-5-prevents-destructive-action-on-production-db.html) about destroying our "production" database.  

I figured that is a good warning to have, so making a staging environment that mirrors all the production settings.  I used this [blog post](http://nts.strzibny.name/creating-staging-environments-in-rails/) as a guide

ultrasaurus commented:
setting the rack timeout with an initializer caused an error calling `heroku pg:reset` -- seems like the code was using old syntax.  The default for the [rack-timeout gem](https://github.com/heroku/rack-timeout) is documented as 15 seconds, which seems like it should be long enough, so just removing the initializer for now.

siruguri commented:
Is this so someone can potentially set up a production-type environment somewhere and test it out? If so, would writing this note somewhere in the README be helpful to future contributors?

I'd say the PR looks fine to me, o/w. HTH!

ultrasaurus commented:
Actually, this was so it would be easier (safer) to destroy data on real staging env: https://github.com/railsbridge/bridge_troll/wiki/Deploying-the-App

Adding a pointer to the README is a good idea -- regular contributors may want to make their own staging environments and might find those instructions useful

ultrasaurus commented:
added [note to readme](https://github.com/railsbridge/bridge_troll/commit/ecb1a7431fc4fd5c3d45486895cf417ff9df0775)

rebased from master, and merged

ultrasaurus commented:
for reference, since the rebase seems to have caused the commits to differ:
* https://github.com/railsbridge/bridge_troll/commit/a0b5bf27c36b37f482487efaa7e7b860af10545b
* https://github.com/railsbridge/bridge_troll/commit/f4758647879e4e3dabdabfdfe6ffe05c7a047da4

prolly should have squashed those for merging
